tests/test_sqlite.py

```diff
@@ -1,34 +1,98 @@
-from ms2query.sqlite import create_table_structure, \
-    add_spectra_to_database, get_spectra_from_sqlite
-from ms2query.app_helpers import gather_test_json
 import os
-import json
-from matchms.Spectrum import Spectrum
+import pandas as pd
+import numpy as np
+from matchms import Spectrum
+from ms2query.app_helpers import load_pickled_file
+from ms2query.create_sqlite_database import make_sqlfile_wrapper
+from ms2query.query_from_sqlite_database import \
+    get_tanimoto_score_for_inchikeys, get_spectra_from_sqlite
 
 
-def test_sqlite_functions(tmp_path):
-    """Tests the functions create_table_structure, add_spectra_to_database
-    and get_spectra_from sqlite"""
+def test_sqlite_functions_wrapper(tmp_path):
+    """Tests all sqlite related functions
 
-    column_type_dict = {"spectrum_id": "VARCHAR",
-                        "source_file": "VARCHAR",
-                        "ms_level": "INTEGER",
-                        "key_that_does_not_exist": "VARCHAR"
-                        }
+    These functions are creating a new sqlite file with spectra data and
+    tanimoto scores (create_sqlite_database.py) and functions to retrieve
+    information from the sqlite database.
+    """
     sqlite_file_name = os.path.join(tmp_path, "test_spectra_database.sqlite")
-    spectra_id_list = ['CCMSLIB00000223876', 'CCMSLIB00003138082']
 
-    # Get path to json testfile
-    path_to_json_test_file = gather_test_json("testspectrum_library.json")[0][
-        "testspectrum_library.json"]
+    path_to_test_files_sqlite_dir = os.path.join(
+        os.path.split(os.path.dirname(__file__))[0],
+        'tests/test_files_sqlite')
 
-    create_table_structure(sqlite_file_name, column_type_dict)
-    add_spectra_to_database(sqlite_file_name, path_to_json_test_file)
-    spectra_list = get_spectra_from_sqlite(sqlite_file_name, spectra_id_list)
+    # Creates a temporary sqlite file with test data.
+    make_sqlite_file(sqlite_file_name, path_to_test_files_sqlite_dir)
+    # Test if the correct tanimoto score data is retrieved from the sqlite file
+    get_tanimoto_scores(sqlite_file_name, path_to_test_files_sqlite_dir)
+    # Test if the correct spectrum data is loaded from the sqlite file
+    get_spectrum_data(sqlite_file_name, path_to_test_files_sqlite_dir)
```

Comment:
I think it would be better to rather have different tests for the different functions. Wrapping tests into functions here is only making harder to follow the code (and exceptions if they occur).
So better just define a simple helper function here, e.g.
```python
def get_file_and_path():
    ...
    return sqlite_file_name, path_to_test_files_sqlite_dir
```